deployer/__main__.py

```diff
@@ -46,6 +47,48 @@ def deploy_support(cluster_name):
         with cluster.auth():
             cluster.deploy_support()
 
+
+def deploy_jupyterhub_grafana(cluster_name):
+    """
+    Deploy grafana dashboards for operating a hub
+    """
+
+    # Validate our config with JSON Schema first before continuing
+    validate(cluster_name)
+
+    config_file_path = Path(os.getcwd()) / "config/hubs" / f'{cluster_name}.cluster.yaml'
+    with open(config_file_path) as f:
+        cluster = Cluster(yaml.load(f))
+
+    # If grafana support chart is not deployed, then there's nothing to do
+    if cluster.support:
+        secret_config_file = Path(os.getcwd()) / "secrets/config/hubs" / f"{cluster_name}.cluster.yaml"
+
+        # Read and set GRAFANA_TOKEN from the cluster specific secret config file
+        with decrypt_file(secret_config_file) as decrypted_file_path:
+            with open(decrypted_file_path) as f:
+                config = yaml.load(f)
+
+        # Get the url where grafana is running from the cluster config
+        grafana_url = cluster.support.get("config", {}).get("grafana", {}).get("ingress", {}).get("hosts", {})
+        uses_tls = cluster.support.get("config", {}).get("grafana", {}).get("ingress", {}).get("tls", {})
+        if grafana_url:
+            grafana_url = "https://" + grafana_url[0] if uses_tls else "http://" + grafana_url[0]
+
+            # Use the jupyterhub/grafana-dashboards deployer to deploy the dashboards to this cluster's grafana
+            with tempfile.TemporaryDirectory() as grafana_dashboards:
+                print("Cloning jupyterhub/grafana-dashboards...")
+                subprocess.check_call(
+                    ["git", "clone", "https://github.com/jupyterhub/grafana-dashboards", grafana_dashboards],
+                    env={'GRAFANA_TOKEN': config["grafana_token"]}
```

Comment:
The env var should be passed to the deploy.py call, rather than the git clone call